lib/Attribute.js

```diff
@@ -56,12 +59,12 @@ function Attribute(schema, name, value) {
   else if (this.type.name === 'list'){
 
     if(value.type) {
+      if (value.list === undefined || value.list[0] === undefined){
```

Comment:
Above we use `!value.map`, here we use `value.list === undefined` and `value.list[0] === undefined`. Is there a reason this line isn't just `(!value.list || !value.list[0])`